requirements/requirements-py-3.11.txt

```diff
@@ -42,7 +42,7 @@ debugpy==1.6.7
 decorator==5.1.1
 defusedxml==0.7.1
 deprecation==2.1.0
-disk-objectstore==1.1.0
+disk-objectstore==1.1.1
```

Comment:
```suggestion
disk-objectstore==1.2
```

Reply:
This file (and other requirements below) are initially generated by pip-compile and the version is sepcific to patch version, so it needs to be `1.2.0`.

Reply:
Just a thought in my mind, I think `pip-compile --extra=atomic_tools --extra=docs --extra=notebook --extra=rest --extra=tests --no-annotate --output-file=requirements/requirements-py-3.9.txt pyproject.toml` command to generate this file needs to be run by CI when there are dependencies changes in `project.toml`. Anyway not related to this PR.

Reply:
Ah I see these are lock files

Reply:
Yes or a pre-commit hook that checks diff

Reply:
`pre-commit` ... Seems you didn't success on pushing to move away from it 🤔

Reply:
promise aiida-core 3.0 no pre-commit